docs/multichain.md

```diff
@@ -2,20 +2,190 @@
 
 ## Introduction
 
-TODO
+One of the most important features of Simperby is its multichain
+interoperability. Any DAO (Decentralized Autonomous Organization) built with
+Simperby can control various Dapps (Decentralized Applications) deployed on
+various existing chains, such as Ethereum. This could include a treasury for the
+organization, a token that the organization issues, or any other application
+that the organization wants to build and deploy. This multichain communication
+is performed in a **trustless and verifiable manner**, with reasonable cost and
+efficiency.
 
-## Consensus & Governance
+## Light Client
 
-TODO
+A light client is a special optimization of a blockchain node that is capable of
+verifying transactions or state entries without having the full blockchain
+state. This is also known as Simplified Payment Verification (SPV) in Bitcoin.
+
+The key idea of a light client is that a blockchain is **cryptographically
+meaningful even if we take only the headers** of the chain (note that the block
+body contains transactions).
+
+In Tendermint and Vetomint, the light client has a "last trusted header" as its
+state, and it only accepts a new header if it is actually finalized by the
+consensus, as the next block to the last trusted header. **There is no authority
```

Comment:
Does it mean the validators of a "last trusted header" validate(vote for a new block) a new block?

Reply:
nope. v2 is part of h1, which is finalize by v1, which is part of h0, and so on.